src/vkscript/command_parser.cc

```diff
@@ -470,13 +470,36 @@ Result CommandParser::ProcessClear() {
 
 Result CommandParser::ParseValues(const std::string& name,
                                   const DatumType& type,
-                                  std::vector<Value>* values) {
+                                  std::vector<Value>* values,
+                                  bool use_std430_layout = false) {
   assert(values);
 
+  uint32_t row_index = 0;
   auto token = tokenizer_->NextToken();
+  size_t seen = 0;
   while (!token->IsEOL() && !token->IsEOS()) {
     Value v;
 
+    // When |type| is a multi-column matrix, fill more values for its
+    // memory layout.
```

Comment:
Can you add why we fill more values here?

Reply:
done